fake-lmp-device-register

```diff
@@ -58,12 +66,15 @@ extendedKeyUsage=critical, clientAuth
             stdout=subprocess.PIPE,
             stderr=subprocess.PIPE,
         )
-        r.check_returncode()
+        if r.returncode != 0:
+            log.error("ERROR: %s", r.stderr.decode())
+            r.check_returncode()
```

Comment:
I guess @doanac already commented somewhere that it is better to remove `stderr-subprocess.PIPE` above and just do `r.check_return_code()` here.

A rationale? Python tracebacks rock.
You could get the same nicety by using `log.exception`, but that uses more lines of code to achieve the same result.